LayoutTests/imported/w3c/web-platform-tests/css/css-properties-values-api/animation/custom-property-animation-transform-list-multiple-values-expected.txt

```diff
@@ -4,5 +4,5 @@ PASS Animating a custom property of type <transform-list> containing multiple va
 PASS Animating a custom property of type <transform-list> containing multiple values with additivity
 PASS Animating a custom property of type <transform-list> containing multiple values with a single keyframe and additivity
 PASS Animating a custom property of type <transform-list> containing multiple values with iterationComposite
-PASS Animating a custom property of type <transform-list> containing multiple values and with mismatching list lengths
+FAIL Animating a custom property of type <transform-list> containing multiple values and with mismatching list lengths assert_equals: expected "matrix(0, -1.5, 1.5, 0, 50, 0)" but got "matrix(0, 1.5, -1.5, 0, 50, 0)"
```

Comment:
It's entirely possible, even likely I'd say, that this test is incorrect and based on our previous incorrect behavior.